first_timers/first_timers.py

```diff
@@ -112,52 +101,13 @@ def tweet_issues(issues, creds, debug=False):
         title = issue['title']
 
         if len(title) > allowed_title_len:
-            title = title[: allowed_title_len - 1] + ellipse
+            title = title[:allowed_title_len - 1] + ELLIPSE
         else:
             if 'languages' in issue:
-                language_hashTags = ''.join(
-                    ' #{}'.format(lang) for lang in issue['languages']
-                )
+                language_hashTags = ''.join(' #{}'.format(lang) for lang in issue['languages'])
```

Comment:
Same as above, revert this back. It is easier to read it when it is multi line

Reply:
will do it then